src/common/object.ts

```diff
@@ -55,3 +55,4 @@ export const nameRegex = new RegExp(/^(.){1,60}$/);
 export const socialHandleRegex = new RegExp(/^@?([\w-]){1,39}$/i);
 export const handleRegex = new RegExp(/^@?[a-z0-9](\w){2,38}$/i);
 export const descriptionRegex = new RegExp(/^[\S\s]{1,250}$/);
+export const emailRegex = new RegExp(/^[\w-.]+@([\w-]+\.)+[\w-]{2,4}$/g);
```

Comment:
Basic email, wouldn't support emails like `name+site@daily.dev`?

Reply:
Should be ok for this purpose right?

Reply:
Also wouldn't support `name@domain.solutions` (longer TLDs)

Reply:
https://emailregex.com

Reply:
https://github.com/colinhacks/zod/blob/8552233c77426f77d3586cc877f7aec1aa0aa45b/src/types.ts#L599-L600